helm_deploy/hmpps-temporary-accommodation-ui/values.yaml

```diff
@@ -42,7 +42,7 @@ generic-service:
   #     [name of environment variable as seen by app]: [key of kubernetes secret to load]
 
   namespace_secrets:
-    approved-premises-ui:
+    temporary-accommodation-ui:
```

Comment:
These secrets need to be written when you have a moment to show me how please @pezholio